interfaces/builtin/opengl.go

```diff
@@ -91,10 +94,11 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 @{PROC}/driver/prl_vtg rw,
 
 # /sys/devices
-/sys/devices/pci[0-9a-f]*/**/config r,
-/sys/devices/pci[0-9a-f]*/**/revision r,
-/sys/devices/pci[0-9a-f]*/**/{,subsystem_}device r,
-/sys/devices/pci[0-9a-f]*/**/{,subsystem_}vendor r,
+/sys/devices/**pci[0-9a-f]*/**/config r,
+/sys/devices/**pci[0-9a-f]*/**/revision r,
+/sys/devices/**pci[0-9a-f]*/**/{,subsystem_}class r,
+/sys/devices/**pci[0-9a-f]*/**/{,subsystem_}device r,
+/sys/devices/**pci[0-9a-f]*/**/{,subsystem_}vendor r,
```

Comment:
Can you adjust these to be somewhat finer with:
```
/sys/devices/{,*pcie-controller/}pci[0-9a-f]*/**/config r,
/sys/devices/{,*pcie-controller/}pci[0-9a-f]*/**/revision r,
/sys/devices/{,*pcie-controller/}pci[0-9a-f]*/**/{,subsystem_}class r,
/sys/devices/{,*pcie-controller/}pci[0-9a-f]*/**/{,subsystem_}device r,
/sys/devices/{,*pcie-controller/}pci[0-9a-f]*/**/{,subsystem_}vendor r,
```

Reply:
Sure, will do